### ISSUE TITLE
Doesn't work:(

### ISSUE THREAD

**Issue by hesum96** [araghon007/X1nput#36]
Hey dude what's up I hope you're doing well I did everything you asked and tried ver 3.1 and 3.0 but doesnt find any exe from desktop (only nvidia share). I dont know if you will read this but it will be amazing if you could provide a solution or add manual drag and drop. Best regards!

**Comment by araghon007** [comment]
Have you tried running it as administrator as well?

**Comment by araghon007** [comment]
Actually, are you trying to inject it into a running program?

**Comment by hesum96** [comment]
Whats up dude Yes I tried to run as administrator and and tried to replace the ini file in the game folder (gears 5 and resident evil village) but didnt work and when i click save button  in the settings no new file is created and nothing happens. There are five exe game files in the desktop but when i scan, it doesnt find any of them.

**Comment by araghon007** [comment]
This tool only works while the game is running, I realized only now that I should've made that clear in the readme. You'll need to launch the game, then do refresh in the configurator.

**Comment by hesum96** [comment]
Oh shit maybe the program was working afterall. Thanks for the response just a quick question. How do i drag the ini file into the game folder? It doesnt create anything there is only one ini file in the program zip.

**Comment by araghon007** [comment]
The program will copy the ini file automatically

**Comment by araghon007** [comment]
Hey, just want to check up. Did you get it working in the end?

**Comment by hesum96** [comment]
What's up bro I'm not in my city (I'm about to start my military service) as soon as I'm back I'll let you know! 👍

**Comment by getill** [comment]
> Hey, just want to check up. Did you get it working in the end?

Hey there ! I got the same problem and my game is running. I'm replying here just in case you wanted to troubleshoot something 😌

**Comment by araghon007** [comment]
I'm assuming you've also tried running the configurator as administrator. What game are you trying to use it with?

**Comment by getill** [comment]
Yes I tried everything I saw on the readme. I got nothing on the app. I'm trying to run assetto corsa. Maybe it's just not compatible ?

**Comment by araghon007** [comment]
I'll need to rewrite the detection logic. Right now it's only looking for games that use xinput, but there might be games that use dinput instead. For now, you can try enabling Xbox controller support in Steam (but not enhanced features because that isn't implemented in the current version), and then injecting into Steam.

**Comment by getill** [comment]
Oh good to know ! I hope our case helped for your app logic ! I'll give it a try

**Comment by getill** [comment]
Still nothing for me with steam. But I'll wait for an eventual update. Thanks for the project, can't wait to try it out !! 👀

**Comment by araghon007** [comment]
Oh, interesting. If you did enable Xbox controller support in Steam, you **must** inject into Steam, because all input gets routed through it.

**Comment by getill** [comment]
Yes I've done that. As I said, I'll try some things by my side to see if I can resolve the problem 😌

**Comment by hesum96** [comment]
> 

I'm a bit late but looking forward for the update! Did everything couldn't get it to work..If i can support in anyway pls let me know

**Comment by araghon007** [comment]
Oh huh, it should've worked with the dongle. Though supposedly there are different versions of the dongle, they should all work the same. Which controller are you using, exactly?

**Comment by getill** [comment]
I tried with the 2019 and 2021 versions of the controller. All with the same dongle that I bought near 2020.

**Comment by araghon007** [comment]
Does the test function under controller setup work with it, at least?

**Comment by getill** [comment]
Yes, I can get a rumble test with dongle but nothing in game. I mean just the regular rumbles

**Comment by araghon007** [comment]
Hmm, I'll try looking into it then. Thanks for the report

**Comment by getill** [comment]
Thanks for your time !

**Comment by nekun1** [comment]
Hi, when I am trying to inject into Cyberpunk 2077 the program crashes. Running as an administrator. Any ideas?

**Comment by shishimaru90** [comment]
Hmm, the app doesn't find any compatible game, even if run with admin or if the game is open.
